kvmagent/kvmagent/test/shareblock_testsuite/shared_block_plugin_teststub.py

```
@@ -54,7 +54,7 @@ def login(self):
 
     def connect2(self, hostUuid, vgUuid):
         # get block uuid
-        r, o = bash.bash_ro("ls /dev/disk/by-id | grep scsi|awk -F '-' '{print $2}'")
+        r, o = bash.bash_ro("ls /dev/disk/by-id | grep scsi-3 | awk -F '-' '{print $2}'")
```

Ruff (0.15.2)

[warning] Unpacked variable `r` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)